handbook/engineering/code-intelligence/index.md

```diff
@@ -1,5 +1,11 @@
 # Code intelligence team
 
-Vision statement: Code intelligence is as good or better than an IDE in the browser for all code hosts and all languages, including cross-repository definitions and references.
-
 [Roadmap](https://docs.google.com/document/d/1cBsE9801DcBF9chZyMnxRdolqM_1c2pPyGQz15QAvYI/edit#heading=h.dimwsc9ccmwq)
+
+## Vision statement
+
+Code intelligence is as good or better than an IDE in the browser for all code hosts and all languages, including cross-repository definitions and references.
+
+## Tech stack
+
+Go, TypeScript, Postgres and many other languages for supporting LSIF and LSP.
```

Comment:
Is SQLite not used at all anymore (just asking)?

Reply:
https://sourcegraph.com/search?q=sqlite+repo:%5Egithub%5C.com/sourcegraph/sourcegraph%24+&patternType=literal